### PR TITLE
Allow blank issues

### PR DESCRIPTION
**Pull request by ivirshup**
Sometimes the templates just don't fit

### CODE REVIEW

**Comment by flying-sheep** [comment]
in which cases?

**Comment by ivirshup** [comment]
Mostly dev process and tracking issues

**Comment by flying-sheep** [comment]
Which are done by us exclusively, so we can just delete the template.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
-blank_issues_enabled: false
+blank_issues_enabled: true
 contact_links:
   - name: Scanpy Community Forum
     url: https://discourse.scverse.org/
     about: If you have questions about “How to do X”, please ask them here.
+  - name: Blank issue
+    url: https://github.com/scverse/scanpy/issues/new
+    about: For things that don't quite fit elsewhere. Please note that other templates should be used in most cases – this is mainly for use by the developers.
```
Comment (flying-sheep) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
```

**Review by flying-sheep — CHANGES_REQUESTED** [review]
I’d also be OK to just do the first line of the diff, but leave out the ability for users to have a clickable thing.

Then we can just remove the `/choose` from the URL to get a blank form.

I rather press <kbd>Ctrl</kbd>+<kbd>⇐</kbd> → <kbd>⇐</kbd> → <kbd>⏎</kbd> than having to deal with a million blank issues with no reproducible example.

**Review by flying-sheep — APPROVED** [review]
OK, this is relatively hidden and not a green button, so it’s fine

**Pull request merged**